src/components/SubNav/SubNav.test.tsx

```diff
@@ -0,0 +1,284 @@
+      <SubNav
+        primaryActions={
+          <>
+            <SubNavButton id="primary-sub-nav-button-1" buttonType="text">
+              Label Text
+            </SubNavButton>
+            <SubNavButton id="primary-sub-nav-button-2" buttonType="text">
+              Label Text
+            </SubNavButton>
+          </>
+        }
+        secondaryActions={
+          <>
+            <SubNavButton id="secondary-sub-nav-button-1" buttonType="text">
+              Label Text
+            </SubNavButton>
+            <SubNavLink
+              id="secondary-sub-nav-link-1"
+              screenreaderOnlyText="for the NYPL Research Catalog"
+              href="#link1"
+            >
+              Label Text
+            </SubNavLink>
+          </>
+        }
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("passes axe accessibility test with props", async () => {
+    const { container } = render(
+      <SubNav
+        actionBackgroundColor="brand.primary-05"
+        highlightColor="brand.primary"
+        id="sub-nav-id"
+        primaryActions={
+          <>
+            <SubNavButton id="primary-sub-nav-button-1" buttonType="text">
+              Label Text
+            </SubNavButton>
+          </>
+        }
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+});
+
+describe("SubNav Component", () => {
+  it("renders with primary actions", () => {
+    render(
+      <SubNav
+        primaryActions={
+          <>
+            <SubNavButton id="primary-button-1">Primary Button 1</SubNavButton>
+            <SubNavButton id="primary-button-2">Primary Button 2</SubNavButton>
+          </>
+        }
+      />
+    );
+
+    expect(screen.getByText("Primary Button 1")).toBeInTheDocument();
+    expect(screen.getByText("Primary Button 2")).toBeInTheDocument();
+  });
+
+  it("renders with secondary actions", () => {
+    render(
+      <SubNav
+        primaryActions={
+          <>
+            <SubNavButton id="primary-button-1">Primary Button</SubNavButton>
+          </>
+        }
+        secondaryActions={
+          <>
+            <SubNavLink href="#secondary-link">Secondary Link</SubNavLink>
+          </>
+        }
+      />
+    );
+
+    expect(screen.getByText("Primary Button")).toBeInTheDocument();
+    expect(screen.getByText("Secondary Link")).toBeInTheDocument();
+  });
+
+  it("renders with selectedItem highlighted", async () => {
+    render(
+      <SubNav
+        primaryActions={
+          <>
+            <SubNavButton id="primary-button-1" isSelected>
+              Primary Button 1
+            </SubNavButton>
+            <SubNavButton id="primary-button-2">Primary Button 2</SubNavButton>
+          </>
+        }
+      />
+    );
+
+    const selectedButton = await screen.findByRole("button", {
+      name: /Primary Button 1/i,
+    });
+    expect(selectedButton).toHaveTextContent("Primary Button 1");
```

Comment:
This test verifies that the component is rendered but not that it's highlighted. You added an extra class name when `isSelected` is passed so check for that instead.